server/protocols/jmap/src/main/java/org/apache/james/jmap/methods/SetMessagesCreationProcessor.java

```diff
@@ -128,6 +137,7 @@ private void handleCreate(CreationMessageEntry create, Builder responseBuilder,
             validateImplementedFeature(create, mailboxSession);
             validateArguments(create, mailboxSession);
             validateRights(create, mailboxSession);
+            validateIsUserOwnerOfMailboxes(create, mailboxSession);
```

Comment:
why it's not covered by validateRights ?

Reply:
validateRights is just checking that one of the sender is the current user